docs/reference/builder.md

```diff
@@ -754,14 +754,15 @@ This will then be visible from `docker inspect` with the other labels.
 
 ## EXPOSE
 
-    EXPOSE <port> [<port>...]
+    EXPOSE <port> [<port>/<protocol>...]
 
 The `EXPOSE` instruction informs Docker that the container listens on the
-specified network ports at runtime. `EXPOSE` does not make the ports of the
-container accessible to the host. To do that, you must use either the `-p` flag
-to publish a range of ports or the `-P` flag to publish all of the exposed
-ports. You can expose one port number and publish it externally under another
-number.
+specified network ports at runtime. You can also specify if the port uses TCP/UDP,
+and defaults to tcp if the protocol is omitted.
+`EXPOSE` does not make the ports of the container accessible to the host. 
```

Comment:
Nope, no problem. :) Go ahead and do your rewording and I'll commit into this PR with any copyedits that need to be made.